telegram/constants.py

```diff
@@ -136,6 +137,7 @@
 """
 from typing import List
 
+BOT_API_VERSION: float = 5.1
```

Comment:
I think standard is version number as string

Reply:
+1